tests/test_config.py

```diff
@@ -235,20 +244,45 @@ def test_clean_added_user_with_refresh(get_bmo_data, repo, config, test_bmo_user
 
     # Compare the old and new people lists without the new entry, they should match.
     new = sorted(
-        [dict(p) for p in file_config.config["people"] if p["bmo_id"] != 3],
+        [
+            dict(person)
+            for person in file_config.config["people"]
+            if person["bmo_id"] != 3
+        ],
         key=itemgetter("bmo_id"),
     )
-    original = sorted([p for p in config["people"]], key=itemgetter("bmo_id"))
+    original = sorted([person for person in config["people"]], key=itemgetter("bmo_id"))
```

Comment:
Nit: you can remove the inner `[]` and just do `sorted(person for...`, since `sorted` takes an iterator and returns a list.

In the `original = ` case, you can technically just do `sorted(config["people"], key=...`

Reply:
> Nit: you can remove the inner `[]` and just do `sorted(person for...`, since `sorted` takes an iterator and returns a list.

That would result in `E999: SyntaxError: Generator expression must be parenthesized`. However, I could turn it into
```python
sorted((person for person in config["people"]), key=itemgetter("bmo_id")).
```
Not sure that is better than the existing form.

> In the `original = ` case, you can technically just do `sorted(config["people"], key=...`

In this case -- 100%. Not sure why I even did this in the first place. It could have been because I was thinking of `file_config.config["people"]`, which actually is not a list, but a `ruamel.yaml.comments.CommentedSeq` and I wanted to convert that to a proper list.

In any case, this is moot here, will change it to `sorted(config["people"], ...`.

Reply:
Ahhhh the `key` kwarg doesn't allow you to use a generator without explicit parenthesis. I suppose this would apply to all functions. TIL. :)